index.html

```diff
@@ -0,0 +1,159 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>☯️ Conference </title>
+    <link rel="stylesheet" href="style.css" />
+    <link
+      rel="stylesheet"
+      href="https://pro.fontawesome.com/releases/v5.10.0/css/all.css"
+      integrity="sha384-AYmEC3Yw5cVb3ZcuHtOA93w35dYTsvhLPVnYs9eStHfGJvOvKxVfELGroGkvsg+p"
+      crossorigin="anonymous"
+    />
+  </head>
+  <body>
+    <section class="grid-container">
+      <section class="Section-1">
+        <div class="social-icons">
+          <a href="#"><i class="fab fa-twitter"></i></a>
+          <a href="#"><i class="fab fa-facebook"></i></a>
+          <ul class="social-lists">
+            <li><a href="#">English</a></li>
+            <li><a href="#">MyPage</a></li>
+            <li><a href="#">Logout</a></li>
+          </ul>
+        </div>
+      </section>
+      <section class="Section-2">
+        <div class="header">
+          <img src="images/logo.png" alt="" class="logo" />
+          <nav class="nav-bar">
+            <ul class="nav-menu">
+              <li class="nav-item">
+                <a href="About.html" class="nav-link">About</a>
+              </li>
+              <li class="nav-item">
+                <a href="#" class="nav-link">Program</a>
+              </li>
+              <li class="nav-item">
+                <a href="#" class="nav-link">Join</a>
+              </li>
+              <li class="nav-item">
+                <a href="#" class="nav-link">Sponsor</a>
+              </li>
+              <li class="nav-item">
+                <a href="#" class="nav-link">News</a>
+              </li>
+              <li><a href="#" class="contact">Contact us</a></li>
+            </ul>
```

Comment:
There are some disparities in the design of the navbar as indicated below. The bottom bar should be white with font color of black. The menu items should also be aligned to the right with logo aligned to the left. Kindly check and add the social media links also 

Your design            |  The design template
:-------------------------:|:-------------------------:
![Screenshot from 2022-10-27 13-18-50](https://user-images.githubusercontent.com/65233860/198296179-53c86b6f-4252-4e94-aaac-9a53c41bf7b7.png)  |  ![Screenshot from 2022-10-27 13-19-11](https://user-images.githubusercontent.com/65233860/198296204-a1c70e29-df6a-4bd8-80f2-46cf02f45981.png)